guides/release/models/finding-records.md

```diff
@@ -2,34 +2,36 @@ The EmberData store provides an interface for retrieving records of a single typ
 
 ### Retrieving a Single Record
 
-Use [`store.findRecord()`](https://api.emberjs.com/ember-data/release/classes/Store/methods/findRecord?anchor=findRecord) to retrieve a record by its type and ID.
-This will return a promise that fulfills with the requested record:
+Use [`findRecord()`](https://api.emberjs.com/ember-data/5.3/functions/@ember-data%2Fjson-api%2Frequest/findRecord) to retrieve a record by its type and ID.
+This will return a single `record` or a `array` depends upon the response from server:
 
 ```javascript
-// GET /blog-posts/1
-this.store.findRecord('blog-post', 1)  // => GET /blog-posts/1
-  .then(function(blogPost) {
-      // Do something with `blogPost`
-  });
+// GET /blog-post/1
+import { service } from '@ember/service';
+import { findRecord } from '@ember-data/json-api/request';
+
+// somewhere in the app
+const result = await this.store.request(findRecord('blog-post', '1'));
```

Comment:
```suggestion
const result = await store.request(findRecord('blog-post', '1')); 
```

I think @runspired mentioned some time ago that we need to move from referencing `this.store` to just `store`

Reply:
Oh, My bad I might have missed it. Thanks for pointing it out. 

 Fixed it now.